src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -155,6 +224,9 @@ START_SECTION(getParameters())
   Param params = ptr->getParameters();
   TEST_EQUAL(params.getValue("integration_type"), INTEGRATION_TYPE_INTENSITYSUM)
   TEST_EQUAL(params.getValue("baseline_type"), BASELINE_TYPE_BASETOBASE)
+  TEST_EQUAL(params.getValue("print_debug"), 0)
```

Comment:
I think the use of the parameter to specify the verbosity is fine for now.  Like you said, we can decide later to remove it if need be.